red_blue_world/patches/pickyeater.py

```diff
@@ -130,7 +128,7 @@ def reset_fruit(self):
     def check_fruit_resetting(self):
         non_rewarding = True
         for [x, y] in self.rewarding_blocks:
-            object_idx = self.object_coords.index((x, y))
+            object_idx = self.object_coords.index([x, y])
```

Comment:
I realize, now, that this is going to be ~`O(N^2)` because `index` is performing a search. Ideally, we'd rework this data structure so that we can have constant time lookups and make this `O(N)`.

This isn't new to this PR, so I'll open an issue to solve it later